browse/templates/abs/extra_services.html

```diff
@@ -52,20 +52,16 @@
   <li><a href="{{url_for('dissemination.pdf', arxiv_id=requested_id)}}" aria-describedby="download-button-info" accesskey="f" class="abs-button download-pdf">Download PDF</a></li>
       {%- endif -%}
       {%- if format == 'latexml' -%}
-  <li>
-    <a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a>
-  </li>
+  <li><a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a></li>
       {%- endif -%}
       {%- if format == 'src' and current-%}
-  <li><a href="{{url_for('src.e_print', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
+  <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
       {%- elif format.startswith('ps') -%}
   <li><a href="{{url_for('dissemination.ps', arxiv_id=requested_id)}}" class="abs-button download-ps">PostScript{% if format.endswith('(400)') %} (400 dpi){% elif format.endswith('(600)') %} (600 dpi){% elif format.endswith('(cm)') %} (Type I cm){% elif format.endswith('(CM)') %} (Type I CM){% endif %}</a></li>
       {%- elif format == 'dvi' -%}
   <li><a href="{{url_for('dissemination.dvi', arxiv_id=requested_id)}}" class="abs-button download-dvi">{{ format.upper() }}</a></li>
       {%- elif format == 'html' -%}
   <li><a href="{{url_for('dissemination.html', arxiv_id=requested_id)}}" accesskey="f" class="abs-button download-html">{{ format.upper() }}</a></li>
-      {%- elif format == 'other' -%}
```

Comment:
This removal is not in master or develop. I'm not sure why adding the HTML should remove the "Other Formats" button.